src/pages/home/index.jsx

```diff
@@ -30,7 +32,8 @@ export default function () {
                         <p className="main_slogan">{dataSource.home.banner.slogan}</p>
 
                         <div className="button_row center">
-                            <a href="https://seatunnel.apache.org/docs/category/start" className="corner_button blue_fill">{dataSource.common.getStart}</a>
+                            {/* TODO next release should be change to /category/start */}
+                            <a href={'/docs/' + versions[0] + '/introduction'} className="corner_button blue_fill">{dataSource.common.getStart}</a>
```

Comment:
I make it jump to our latest release version docment

Reply:
But we have to change it to `/category/start` in the next release docs, so I add some todo mark here

Reply:
ok